tsfc/fem.py

```diff
@@ -28,7 +29,12 @@
 
 
 class ReplaceSpatialCoordinates(MultiFunction):
+    """Replace SpatialCoordinate nodes with the ReferenceValue of a
+    Coefficient.  Assumes that the coordinate element only needs
+    affine mapping.
```

Comment:
One could assert this by doing `assert o.ufl_coordinate_element().mapping() == "identity"` in the `spatial_coordinate` handler.